### PR TITLE
Added a few ++ formatting and spelling

### PR DESCRIPTION
**Pull request by elementalvoid**

### CODE REVIEW

**Comment by woneill** [comment]
I'm assigning this to the man himself for confirmation. Which makes me think I should make him the CODEOWNER too....

File: README.md [hunk 1]
```diff
@@ -16,28 +16,35 @@
...
-* Fried Eggs on Burgers
-* Hipchat
+* HipChat
 * Trend IMSS
+* Joe Nacchio
+* Ubuntu
+* Complainy Hulu customers
+* Places called "Fish Tacos"
+* Red ants
+* Tulsa, OK
```
Comment (rpkish) [inline README.md]: These are not Kishtonites, I think this was bleed over from another list.

**Review by rpkish — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (woneill) [reply README.md]: How many lists does Kish have?!?!?
conspiracy

**Review by rpkish — APPROVED** [review]
Thanks for the corrections and additions!
+1

File: README.md [hunk 2]
```diff
@@ -16,28 +16,33 @@
...
 * Pearl Jam
-* Fried Eggs on Burgers
-* Hipchat
+* HipChat
 * Trend IMSS
+* Joe Nacchio
+* Ubuntu
+* Complainy Hulu customers
+* Places called "Fish Tacos"
+* Going south of Alameda
```
Comment (kr3cj) [inline README.md:48]: 😆

**Review by kr3cj — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (elementalvoid) [reply README.md]: You know .. I saw those and thought "huh, Kish doesn't like those either?"

Looking at it again, that would appear to be @kr3jc's budding attempt at a Klichtonite (me) list.

Comment (kr3cj) [reply README.md]: ^That. Don't forget "Hulu customers that complain about ads" for whenever we create `Klichtonite`

**Pull request merged**